File: deaf_grandma.py
def deaf_grandma():
    count = 0
    prompt = input("Say hello to MeeMaw!\n")
    while count < 2:
        if prompt == '':
            prompt = input("WHAT?!\n")
        elif prompt != prompt.upper():
            prompt = input("SPEAK UP, KID!\n")
        elif prompt == "GOODBYE!":
            if count == 1:
                print("LATER, SKATER!")
                return
            else: prompt = input("LEAVING SO SOON?!\n")
            count += 1
        elif prompt == prompt.upper():
            prompt = input("NO, NOT SINCE 1946!\n")

deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    let count = 0
    let prompt = window.prompt("Say hello to MeeMaw!")
    while(count < 2){
        if(prompt === ''){
            prompt = window.prompt("WHAT?!")
        }

        else if(prompt !== prompt.toUpperCase()){
            prompt = window.prompt("SPEAK UP, KID!") 
        }

        else if(prompt === 'GOODBYE!'){
            if(count === 1){
                window.alert("LATER, SKATER!")
                return
            }
            else prompt = window.prompt("LEAVING SO SOON?!")
            count++
            }

        else if(prompt === prompt.toUpperCase()){
            prompt = window.prompt("NO, NOT SINCE 1946!")
        }
    }
}

deafGrandma();